index.html

```diff
@@ -96,24 +124,23 @@ <h2 id="docconformance">
         The following table provides normative per-element document-conformance
         requirements for the use of ARIA markup in HTML documents and describes
         the <a>implicit ARIA semantics</a> that apply to <a data-cite=
-        "html/dom.html#elements">HTML elements</a> as defined in the HTML
-        Accessibility API Mappings 1.0 [[html-aam-1.0]] specification.
+        "html/dom.html#elements">HTML elements</a> as defined in the [[html-aam-1.0|HTML
+        Accessibility API Mappings 1.0 ]] specification.
       </p>
       <p>
-        Each language feature (element or attribute) in a cell in the first column
-        implies the ARIA semantics (any role, states, and properties) given in
-        the cell in the second column of the same row. The third cell in each
-        row defines which ARIA role values and aria-* attributes which MAY be
-        used. Where a cell in the third column includes the term
-        <dfn><strong>Any</strong> `role`</dfn> it indicates that any `role`
-        value apart from the <a>implicit ARIA semantics</a> `role` value, MAY
-        be used. If an cell in the third column includes the term <dfn><strong>No `role`</strong></dfn> it indicates that authors MUST NOT overwrite the implicit ARIA semantics, or native semantics of the HTML element.
+        Each language feature (element or attribute) in a cell in the first column implies the
+        ARIA semantics (any role, states, and properties) given in the cell in the second column
+        of the same row. The third cell in each row defines which ARIA role values and `aria-*`
+        attributes which MAY be used. Where a cell in the third column includes the term
+        <dfn><strong>Any</strong> `role`</dfn> it indicates that any `role` value apart from the
+        <a>implicit ARIA semantics</a> `role` value, MAY be used. If a cell in the third column
+        includes the term <dfn><strong>No `role`</strong></dfn> it indicates that authors MUST NOT
+        overwrite the implicit ARIA semantics, or native semantics of the HTML element.
       </p>
 
       <p class="note" id="aria-usage-note">
-        Setting an ARIA `role` and/or `aria-*` attribute that matches the
-        <span>implicit ARIA semantics</span> is unnecessary and is NOT
-        RECOMMENDED as these properties are already set by the browser.
+        While setting an ARIA `role` and/or `aria-*` attribute that matches the
+        <span>implicit ARIA semantics</span> is NOT RECOMMENDED, in some situations explicitly setting these attributes can be helpful. For instance, if supporting user agents which lack specific implicit semantics.
```

Comment:
"if supporting user agents" > "in supporting user agents"

Reply:
maybe even drop the "in supporting", because if the user agents were "supporting", they'd HAVE / expose those implicit semantics. so just "in user agents which lack..."